src/lib/transform.js

```diff
@@ -286,6 +355,11 @@ function makeGoalsAndObjectivesObject(objectiveRecords) {
       // Make sure its not another objective for the same goal.
       if (goalIds[goalName] && !goalIds[goalName].includes(goalId)) {
         accum[`goal-${existingObjectiveTitle}-id`] = `${accum[`goal-${existingObjectiveTitle}-id`]}\n${goalId}`;
+        accum[`goal-${goalNum}-source`] = uniq([...accum[`goal-${goalNum}-source`].split('\n'), goal.source]).join('\n');
+        if (goal.isCurated) {
+          accum[`goal-${goalNum}-fei-root-causes`] = `${accum[`goal-${goalNum}-fei-root-causes`]}\n${goal.responses
```

Comment:
is there a reason this is not also running uniq?

Reply:
good call